packages/lesswrong/lib/collections/posts/schema.tsx

```diff
@@ -42,6 +42,7 @@ import { stableSortTags } from '../tags/helpers';
 import { getLatestContentsRevision } from '../revisions/helpers';
 import { marketInfoLoader } from './annualReviewMarkets';
 import { getUnusedSlugByCollectionName } from '@/lib/helpers';
+import { CurationNotices } from '../curationNotices/collection';
```

Comment:
remove this import (importing collection files into other collection files/schemas could cause problems).  you in any case have access to it from the ResolverContext